recipes/cpython/all/conanfile.py

```diff
@@ -163,14 +170,14 @@ def package_id(self):
         del self.info.options.env_vars
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
+        get(self, **self.conan_data["sources"][self.version],
                   destination=self._source_subfolder, strip_root=True)
 
     @property
     def _with_libffi(self):
         # cpython 3.7.x on MSVC uses an ancient libffi 2.00-beta (which is not available at cci, and is API/ABI incompatible with current 3.2+)
         return self._supports_modules \
-               and (self.settings.compiler != "Visual Studio" or tools.Version(self._version_number_only) >= "3.8")
+               and (not is_msvc(self) or Version(self._version_number_only) >= "3.8")
```

Comment:
Missing the import. And looking at the code there's a bunch h of places its needed